include/rocksdb/env.h

```diff
@@ -282,7 +282,8 @@ class Env {
   virtual Status FileExists(const std::string& fname) = 0;
 
   // Store in *result the names of the children of the specified directory.
-  // The names are relative to "dir".
+  // The names are relative to "dir", and will never include the
```

Comment:
I prefer "shall" to "will" when there is an expectation placed on user code (in addition to providing a guarantee).

Reply:
As a native English speaker I was under the impression that the use of `shall` had fallen out of favour; especially within American English. I am certainly no expert in English, but various sources I checked seem to confirm that. Do you still want that change made?

Reply:
Wikipedia doesn't lie, right? ;)

https://en.wikipedia.org/wiki/Shall_and_will#Legal_and_technical_use

> Shall and will are distinguished by NASA[18] and Wikiversity[19] as follows:
>
> * Shall is usually used to state a device or system's requirements. For example: "The selected generator shall provide a minimum of 80 Kilowatts."
> * Will is generally used to state a device or system's purpose. For example, "The new generator will be used to power the operations tent."

Reply:
FWIW, standards documents use "shall" or "shall not" to indicate requirements.

Reply:
@anand1976 I was aware of MUST, COULD, SHOULD, MAY... but not SHALL, e.g. https://tools.ietf.org/html/rfc2119

Reply:
@pdillinger I have changed it to `shall` as requested.